### PR TITLE
Lw balance get

### PR DESCRIPTION
**Pull request by lindawoo-plaid**
1) add balance/get endpoint and display current and available balances in UI
2) change accounts_table so that it is updatable (remove join with items table)
3) change createAccounts to createAccount so that when an item is linked, only one account is created and added to db
4) add transferFunds page
5) add IS_PROCESSOR variable to .env file and add processor/token/create endpoint

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -10,6 +10,10 @@ PLAID_SECRET_SANDBOX=
 
 PLAID_ENV=sandbox
 
+# If you want to test the app using a payment processor, set PROCESSOR to 'true'
```
Comment (otherchen) [inline .env.template:13]: do we know which payment processor this would cause the user to use?

File: client/src/components/TransferFunds.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import { NumberInput } from 'plaid-threads/NumberInput';
+import { Button } from 'plaid-threads/Button';
+import { currencyFilter } from '../util';
+
+interface Props {}
+const TransferFunds: React.FC<Props> = (props: Props) => {
+  const [transferAmount, setTransferAmount] = useState('');
+  const handleSubmit = (e: any) => {
+    e.preventDefault();
+    // checkAmount(); logic to check if amount is less than balance/get available balance
```
Comment (otherchen) [inline client/src/components/TransferFunds.tsx:12]: this is just an informative comment to let users know they should implement this function right?

I wonder if we should be more clear here - `You should implement this logic to check if...`

File: client/src/components/TransferFunds.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+
+import { NumberInput } from 'plaid-threads/NumberInput';
+import { Button } from 'plaid-threads/Button';
+import { currencyFilter } from '../util';
+
+interface Props {}
+const TransferFunds: React.FC<Props> = (props: Props) => {
+  const [transferAmount, setTransferAmount] = useState('');
```
Comment (otherchen) [inline client/src/components/TransferFunds.tsx:9]: I'm wondering why we use a string here. Shouldn't transferAmount be a numeric value?

File: server/routes/linkTokens.js [hunk 4]
```diff
@@ -49,7 +49,7 @@ router.post(
         access_token: accessToken,
       };
       // If user has entered a redirect uri in the .env file
-      if (redirect_uri.length > 1) {
+      if (redirect_uri.indexOf('http') === 0) {
```
Comment (otherchen) [inline server/routes/linkTokens.js:52]: I'm wondering if we should error when the redirect_uri is formatted incorrectly so the user knows what went wrong and how to fix it.

File: server/webhookHandlers/handleTransactionsWebhook.js [hunk 5]
```diff
@@ -1,201 +0,0 @@
-/**
```
Comment (otherchen) [inline server/webhookHandlers/handleTransactionsWebhook.js:1]: was this file not being used anymore?

**Review by otherchen — COMMENTED** [review]
Looks great so far! Left a few questions :)

File: .env.template (hunk 1 above)
Comment (lindawoo-plaid) [reply .env.template:13]: I will put in a note that it is Dwolla for the sample app

File: client/src/components/TransferFunds.tsx (hunk 2 above)
Comment (lindawoo-plaid) [reply client/src/components/TransferFunds.tsx:12]: this is a placeholder for me to add logic later...

File: client/src/components/TransferFunds.tsx (hunk 3 above)
Comment (lindawoo-plaid) [reply client/src/components/TransferFunds.tsx:9]: the numberInput in Threads takes a string!!!

File: server/routes/linkTokens.js (hunk 4 above)
Comment (lindawoo-plaid) [reply server/routes/linkTokens.js:52]: will make a note to do this in the future

File: server/webhookHandlers/handleTransactionsWebhook.js (hunk 5 above)
Comment (lindawoo-plaid) [reply server/webhookHandlers/handleTransactionsWebhook.js:1]: nope. no transactions used in account funding app

**Pull request merged**